perf/load-incluster/helm/client/templates/client.yaml

```diff
@@ -0,0 +1,28 @@
+{{range $i := until (int .Values.entries)}}
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: fortio-cli-{{$i}}
+spec:
+  replicas: 1
```

Comment:
I'm curious how this compares to just varying the number of replicas

Reply:
sure.  Changing replicas changes the number of fortio deployments but not the number of services.  By changing the number of services, there is a unique mapping from client to server deployment.  The goal (I had) was to create a large number of services in addition to endpoints with real load, and then expand the workload and see how Istio holds up.  (To mostly match Michael's test case with IRS).